docs/topics/java-read-scanner.md

```diff
@@ -0,0 +1,70 @@
+[//]: # (title: Standard input)
+
+> Java Scanner is a slow tool. Use it only when you need the specific functionalities it offers.
+> Otherwise, it's generally preferable to use Kotlin's `readln()` function to [read standard input](basic-syntax.md#read-from-the-standard-input).
+>
+{type="note"}
+
+In Kotlin, you can use Java Scanner to read data from standard input. Due to Kotlin's interoperability with Java libraries,
+it's possible to access Java Scanner from Kotlin code out of the box.
```

Comment:
The pages in this section compare how things are done in Java and how - in Kotlin (usually more convenient and terse). So it would be natural to compare here what alternatives to using `Scanner` exist.